go.mod

```diff
@@ -4,33 +4,27 @@ go 1.12
 
 require (
 	github.com/blang/semver v3.5.0+incompatible
-	github.com/certifi/gocertifi v0.0.0-20180905225744-ee1a9a0726d2 // indirect
-	github.com/getsentry/raven-go v0.0.0-20190513200303-c977f96e1095 // indirect
-	github.com/ghodss/yaml v0.0.0-20150909031657-73d445a93680
-	github.com/jteeuwen/go-bindata v0.0.0-00010101000000-000000000000
-	github.com/openshift/api v0.0.0-20191001124347-8033e226059b
-	github.com/openshift/client-go v0.0.0-20191001081553-3b0e988f8cb0
-	github.com/openshift/library-go v0.0.0-20190927184318-c355e2019bb3
-	github.com/pkg/profile v1.3.0 // indirect
-	github.com/prometheus/client_golang v0.9.2
+	github.com/ghodss/yaml v1.0.0
+	github.com/jteeuwen/go-bindata v3.0.8-0.20151023091102-a0ff2567cfb7+incompatible
+	github.com/openshift/api v0.0.0-20191217141120-791af96035a5
+	github.com/openshift/client-go v0.0.0-20191216194936-57f413491e9e
+	github.com/openshift/library-go v0.0.0-20191218095328-1c12909e5923
+	github.com/prometheus/client_golang v1.1.0
 	github.com/spf13/cobra v0.0.5
-	github.com/spf13/pflag v1.0.3
-	github.com/xlab/handysort v0.0.0-20150421192137-fb3537ed64a1 // indirect
-	k8s.io/api v0.0.0-20190918155943-95b840bb6a1f
-	k8s.io/apiextensions-apiserver v0.0.0-20190918161926-8f644eb6e783 // indirect
-	k8s.io/apimachinery v0.0.0-20191004115801-a2eda9f80ab8
-	k8s.io/client-go v0.0.0-20191004120905-f06fe3961ca9
-	k8s.io/component-base v0.0.0-20190918160511-547f6c5d7090
-	k8s.io/klog v0.4.0
-	k8s.io/kube-aggregator v0.0.0-20190918161219-8c8f079fddc3 // indirect
-	vbom.ml/util v0.0.0-20180919145318-efcd4e0f9787 // indirect
+	github.com/spf13/pflag v1.0.5
+	k8s.io/api v0.17.0
+	k8s.io/apimachinery v0.17.0
+	k8s.io/client-go v0.17.0
+	k8s.io/component-base v0.17.0
+	k8s.io/klog v1.0.0
 )
 
 replace (
 	github.com/jteeuwen/go-bindata => github.com/jteeuwen/go-bindata v3.0.8-0.20151023091102-a0ff2567cfb7+incompatible
 	github.com/openshift/api => github.com/openshift/api v3.9.1-0.20191209132752-992bc3a41fe6+incompatible
 	github.com/openshift/client-go => github.com/openshift/client-go v0.0.0-20191205152420-9faca5198b4f
-	github.com/openshift/library-go => github.com/openshift/library-go v0.0.0-20191211124107-e0f1590a316e
+	github.com/openshift/library-go => github.com/openshift/library-go v0.0.0-20191218095328-1c12909e5923
+	github.com/prometheus/client_golang => github.com/prometheus/client_golang v0.9.2
```

Comment:
pinned the prometheus client back to avoid incompatible changes

Reply:
Yeah, I think @sallyom will get to it when she brings new k8s version.